src/OrchardCore/OrchardCore.Abstractions/Setup/SetupConstants.cs

```diff
@@ -0,0 +1,15 @@
+namespace OrchardCore.Abstractions.Setup
+{
+    public static class SetupConstants
+    {
```

Comment:
I'm not a big fan of having static classes for constants.

It forces modules that really shouldn't need to, to take references to projects, where a Magic String would have sufficed.

I think @hisham you might be a bit obsessed with the idea that magic strings are bad, where Orchard Core uses them a lot, for the reasons of avoiding references where not necessary, and leaving the code more readable.

We don't attract bugs from this, we have had more issues where people have used the wrong constant, because it becomes less clear in the code, what the constant means.

The whole point of this change, was to move to a properties dictionary, where we don't need to update the code to change a property. Now we end up with that, but have a class where we have to update the code to change the property.

I'm not going to argue against it strongly, but I'm not a fan.